backend/app/api/endpoints/groups.py

```
@@ -208,6 +223,221 @@ def list_members(
     return result
 
 
+@router.get(
+    "/{group_name:path}/entity-members",
+    response_model=list[GroupEntityMemberResponse],
+)
+def list_entity_members(
+    group_name: str = Path(
+        ..., description="Group name (may contain slashes for subgroups)"
+    ),
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)